docs/content/ingestion/index.md

```diff
@@ -173,6 +175,11 @@ handle all formatting decisions on their own, without using the ParseSpec.
 | dimensionExclusions | JSON String array | The names of dimensions to exclude from ingestion. | no (default == [] |
 | spatialDimensions | JSON Object array | An array of [spatial dimensions](../development/geo.html) | no (default == [] |
 
+## Rollup
```

Comment:
@kaijianding can we remove this section and have this info in the description of rollup in the table above?